delegate-app/Main.hs

```diff
@@ -17,32 +17,32 @@ import Plutus.V2.Ledger.Api (POSIXTime (..))
 
 import HydraAuction.Delegate (
   DelegateEvent (..),
-  DelegateInput (DelegateEvent),
   DelegateRunnerT,
-  delegateStep,
+  delegateEventStep,
   execDelegateRunnerT,
  )
 import HydraAuction.Delegate.Interface (DelegateResponse (..))
 import HydraAuction.OnChain.Common (secondsLeftInInterval, stageToInterval)
 import HydraAuction.Tx.Common (currentAuctionStage, currentTimeMilliseconds)
 
-consumer :: Chan DelegateInput -> DelegateRunnerT IO ()
-consumer delegateInputs = do
+consumer :: Chan DelegateEvent -> DelegateRunnerT IO ()
+consumer delegateEvents = do
   -- This will block until an event appears in the queue
-  delegateIn <- liftIO $ readChan delegateInputs
-  delegateResponse <- delegateStep delegateIn
-  liftIO $ mbQueueAuctionPhases delegateResponse delegateInputs
+  -- FIXME: support frontend requests too
+  delegateEvent <- liftIO $ readChan delegateEvents
+  delegateResponse <- delegateEventStep delegateEvent
```

Comment:
Maybe we could remove the return type from `delegateEventStep`, and move `mbQueueAuctionPhases` to the frontend request handler then. We don't really need to respond to anything when handling delegate events.

Reply:
I only changed Main to pass CI, @MangoIV PR #217 gonna rewrite Main.

> We don't really need to respond to anything when handling delegate events.

That is not correct, we need to broadcast things from `delegateEventStep`. Like New bid or Auction state change, for example.